gateway/proxy/proxy/httpserver.py

```diff
@@ -0,0 +1,159 @@
+import ssl
```

Comment:
Taking into account that Gateway is a Django project I would maintain `proxy` project outside `gateway` folder with it's own pipeline and so on if we are not using anything from Django.

Reply:
I'll move it to ./proxy.